### ISSUE TITLE
Getting Error -- AccessibilityInfo could not be found within the project.

### ISSUE THREAD

**Issue by NSR88** [wix/react-native-calendars#1389]
## Description

Versions, I am using --
    "react": "^16.13.1",
    "react-native": "^0.63.4",
    "react-native-calendars": "^1.20.0"

When I try to run bundle command for android, I am getting error -- 
**Error: Unable to resolve module `AccessibilityInfo` from `react-native-calendars/node_modules/react-native/Libraries/react-native/react-native-implementation.js`: AccessibilityInfo could not be found within the project.**

## Expected Behavior

It should be bundled successfully

## Observed Behavior
Getting error when bundling 

```
error Unable to resolve module `AccessibilityInfo` from `react-native-calendars/node_modules/react-native/Libraries/react-native/react-native-implementation.js`: AccessibilityInfo could not be found within the project.

If you are sure the module exists, try these steps:
 1. Clear watchman watches: watchman watch-del-all
 2. Delete node_modules: rm -rf node_modules and run yarn install
 3. Reset Metro's cache: yarn start --reset-cache
 4. Remove the cache: rm -rf /tmp/metro-*. Run CLI with --verbose flag for more details.```

## Environment

Please run these commands in the project folder and fill in their results:

* `npm ls react-native-calendars`: 
    MyApp@1.0.2 /Users/narendrarathore/Documents/git/myapp_minimal_working/myapp
└── react-native-calendars@1.20.0 -> /Users/narendrarathore/Documents/git/myapp_minimal_working/myapp/react-native-calendars

* `npm ls react-native`:

MyApp@1.0.2 /Users/narendrarathore/Documents/git/myapp_minimal_working/myapp
├─┬ @react-native-community/async-storage@1.12.1
│ └── react-native@0.63.4 deduped
├─┬ @react-native-community/datetimepicker@3.0.9
│ └── react-native@0.63.4 deduped
├─┬ @react-native-community/netinfo@5.9.9
│ └── react-native@0.63.4 deduped
├─┬ @react-native-community/viewpager@4.2.2
│ └── react-native@0.63.4 deduped
├─┬ @react-native-firebase/app@10.4.0
│ └── react-native@0.63.4 deduped
├─┬ @react-native-picker/picker@1.9.8
│ └── react-native@0.63.4 deduped
├─┬ native-base@2.8.0
│ ├─┬ jest-react-native@18.0.0
│ │ └── react-native@0.63.4 deduped
│ ├─┬ native-base-shoutem-theme@0.2.2
│ │ └── react-native@0.63.4 deduped
│ ├─┬ react-native-keyboard-aware-scroll-view@0.5.0
│ │ ├─┬ react-native-iphone-x-helper@1.1.0
│ │ │ └── react-native@0.63.4 deduped
│ │ └── react-native@0.63.4 deduped
│ └── react-native@0.63.4 deduped
├─┬ react-native-android-location-services-dialog-box@2.8.2
│ └── react-native@0.63.4 deduped
├─┬ react-native-app-settings@2.0.0
│ └── react-native@0.63.4 deduped
├─┬ react-native-calendars@1.20.0 -> /Users/narendrarathore/Documents/git/myapp_minimal_working/myapp/react-native-calendars
│ └── react-native@0.47.2
├─┬ react-native-check-box@2.1.3
│ └── react-native@0.63.4 deduped
├─┬ react-native-document-picker@3.5.3
│ └── react-native@0.63.4 deduped
├─┬ react-native-fused-location@0.5.1
│ └── react-native@0.63.4 deduped
├─┬ react-native-modal@11.4.0
│ └── react-native@0.63.4 deduped
├─┬ react-native-router-flux@4.0.1
│ ├── react-native@0.63.4 deduped
│ └─┬ react-navigation@2.13.0
│   ├─┬ react-native-safe-area-view@0.9.0
│   │ └── react-native@0.63.4 deduped
│   ├── react-native@0.63.4 deduped
│   ├─┬ react-navigation-deprecated-tab-navigator@1.3.0
│   │ ├─┬ react-native-tab-view@0.0.77
│   │ │ └── react-native@0.63.4 deduped
│   │ └── react-native@0.63.4 deduped
│   ├─┬ react-navigation-drawer@0.5.0
│   │ ├─┬ react-native-drawer-layout-polyfill@1.3.2
│   │ │ ├─┬ react-native-drawer-layout@1.3.2
│   │ │ │ └── react-native@0.63.4 deduped
│   │ │ └── react-native@0.63.4 deduped
│   │ └── react-native@0.63.4 deduped
│   ├─┬ react-navigation-stack@0.3.0
│   │ └── react-native@0.63.4 deduped
│   └─┬ react-navigation-tabs@0.7.0
│     ├─┬ react-native-tab-view@1.0.2
│     │ └── react-native@0.63.4 deduped
│     └── react-native@0.63.4 deduped
├─┬ react-native-snackbar@2.3.1
│ └── react-native@0.63.4 deduped
├─┬ react-native-typography@1.3.0
│ └── react-native@0.63.4 deduped
├─┬ react-native-webview@11.0.2
│ └── react-native@0.63.4 deduped
└─┬ react-native@0.63.4
  └─┬ @react-native-community/cli@4.13.1
    └── react-native@0.63.4 deduped

1. Phone/emulator/simulator & version:  Android Pie (MI A1)
